app/templates/eventStudentView.html

```diff
@@ -0,0 +1,97 @@
+{% set title = "View Event" %}
+{% extends "base.html"%}
+
+{% block styles %}
+    {{super()}}
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+{% endblock %}
+
+{% block scripts %}
+    {{super()}}
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
+{% endblock %}
+
+{% block app_content%}
+    <div class="text-center mt-5 mb-3">
+        {% set page_title = eventData.name %}
+    <h1 id="title">{{page_title}}</h1>
+    {% if isPastEvent %}
+      <div class="alert alert-danger" role="alert">This event is in the past.</div>
+    {% endif %}
+    </div>
+    {% if isPastEvent %}
+        <p><font size="+2"><b>{{eventData.name}}</b> took place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and was located in <b>{{eventData.location}}</b>.</font></p>
+    {% else %}
+        <p><font size="+2"><b>{{eventData.name}}</b> will take place on <b>{{eventData.startDate}}</b> from <b>{{eventData.timeStart}}</b> to <b>{{eventData.timeEnd}}</b> and will be located in <b>{{eventData.location}}</b>.</font></p>
+    {% endif %}
+            <div>
+                <div>
+                    <font size="+2"><h3><br>Description:</h3>{{eventData.description}}<br><br>
+                    <div>
+                        {% if eventData.isService %}
+                            This event earns service hours.
+                        {% elif not eventData.isService %}
```

Comment:
You can just use an `else` here and below

Reply:
resolved